### ISSUE TITLE
increased support for screen readers

### ISSUE THREAD

**Issue by ninhhoang2005** [zbabac/VCT#6]
hello
after used VCT, i see:
1. Access to screen readers is very limited.
I used NVDA(screen reader for the blind) to interact with this tool but it was very disadvantageous because the approach was not good.
The clearest evidence is that some screen reader buttons cannot read attached labels, or combo boxes cannot arrow up/down but must be opened to be able to select.
2. Improved interface.
The problem I'm seeing is: the interface is terrible because it has too many elaborate details.
Let's significantly improve these interfaces to make the software more beautiful and reduce complex parameters.

Above are my personal opinions for this tool.
Looking forward to hearing from you and your colleagues soon.
Thank you for taking a moment to read.
